src/components/filters/AttributeFilter/AttributeDropdown.tsx

```diff
@@ -277,11 +275,24 @@ export class AttributeDropdownWrapped extends React.PureComponent<
         });
 
         this.dataSource.onChange((result: any) => {
+            const { selection } = this.state;
+
+            const items = result.data.items.map((i: any) => i || { empty: true });
```

Comment:
btw the IAttributeDropdownStateItem has property 'selected' - do you know why is it there? i don't see it being set anywhere by anyone - is it just some leftover crap?

if so, i suggest to mark as @deprecated or straight out delete (it is in an exported type.. that i don't see used anywhere either)

Reply:
good eye, leftover crap removed